maintenance/log/handler.go

```diff
@@ -28,10 +28,18 @@ const RequestIDHeader = "Request-Id"
 // in the request specific Sink.
 func Handler(silentPrefixes ...string) func(http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
-		return hlog.NewHandler(log.Logger)(
+		chain := hlog.NewHandler(log.Logger)(
```

Comment:
I think we can skip the variable assign and just return this if TRUE und else the part that is in the if guard you added, right?

Reply:
Fixed